app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1490,3 +1490,12 @@ $include-html-global-classes: $include-html-classes;
 .button {
   background-color: #01a1dd;
 }
+
+.alert-box {
+  position: absolute;
```

Comment:
Properties should be ordered font-size, opacity, position, text-align, width, z-index